drivers/adc/adc_mcux_lpadc.c

```diff
@@ -195,7 +195,8 @@ static void mcux_lpadc_start_channel(const struct device *dev)
 	lpadc_conv_command_config_t cmd_config;
 
 	LPADC_GetDefaultConvCommandConfig(&cmd_config);
-	cmd_config.channelNumber = data->channel_id;
+	cmd_config.channelNumber = data->channel_id & 0b111;
+    cmd_config.sampleChannelMode = (data->channel_id & 8) ? kLPADC_SampleChannelSingleEndSideB : kLPADC_SampleChannelSingleEndSideA;
```

Comment:
Some remarks:
* Use tabs for indentation
* Split into multiple lines, as this crosses the maximum line length
* Use hex values `0x07` and `0x08` instead of mixing for binary operators